src/extra/AutoUI/index.tsx

```diff
@@ -269,6 +269,7 @@ export const AutoUI = <T extends AutoUIBaseResource<T>>({
 	return (
 		<Flex
 			flexDirection={Array.isArray(data) ? 'column' : undefined}
+			flex={1}
 			mt={Array.isArray(data) ? 2 : undefined}
```

Comment:
I think both of these (and all BoxProps tbh) should be wired by the consumer/parent component.